pkg/apis/secret/v0alpha1/secure_value.go

```diff
@@ -39,7 +41,7 @@ type SecureValueSpec struct {
 	// Minimum and maximum lengths in bytes.
 	// +k8s:validation:minLength=1
 	// +k8s:validation:maxLength=24576
-	Value ExposedSecureValue `json:"value,omitempty"`
+	Value common.RawSecureValue `json:"value,omitempty"`
```

Comment:
^^^ moved this helper to common

Reply:
Yes.. we might need to keep that for the moment, I am checking with the cog team to see if there's a way we can reference this new type when generating our Go types, but unsure if we'll be able to.